python/GafferUI/SpreadsheetUI/_Metadata.py

```diff
@@ -191,11 +191,11 @@
 			values are formatted as follows :
 
 			```
-			[
-			    { "name" : "row1Name", "cells" : { "columnName" : columnValue, ... } },
-			    { "name" : "row2Name", "cells" : { "columnName" : columnValue, ... } },
+			{
+			    "row1Name", { "columnName" : columnValue, ... },
+			    "row2Name", { "columnName" : columnValue, ... },
 			    ...
-			]
+			}
```

Comment:
You went back to using a `,` instead of a `:` for a dictionary here :P

Reply:
Gah! I'm a muppet. At least my muppetry is consistent I suppose. Thanks for spotting it - I've force-pushed a fix.